module-2/backend/week_one.md

```diff
@@ -5,30 +5,56 @@ Fork this respository. Answer the questions to the best of your ability. Try to
 Note: When you're done, submit a PR. 
 
 1. List the five common HTTP verbs and what the purpose is of each verb.
+    * Get, post, put, delete, patch
 2. What is Sinatra?
+    * a Web framework that allows us to make web app in ruby
 4. What is MVC?
+    * A model for developing apps in frameworks like rails and sinatra
 5. Why do we follow conventions when creating our actions/path names in our Sinatra routes?
+    * So that others can easily follow our app
 6. What types of variables are accessible in our view templates without explicitly passing them?
+    * instance variables 
 7. Given the following block of code, how would I pass an instance variable `count` with a value of `1` to my `index.erb` template?
   
   ```ruby
   get '/horses' do
+    @count = 1
     erb :index
   end
   ```
 
 8. In the same code block, how would I pass a local variable `name` with a value of `Mr. Ed` to the view?
+    * name = "Mr Ed"
+      erb:index, {locals: {name: name}}
 9. What's the purpose of ERB?
+    * Inserting ruby code into html
 10. Why do I need a development AND test database?
+    * To Not disturb your development database when running tests. 
 11. What's responsive design?
+    * different size devices 
 12. What is CRUD and why is it important?
+    * CRUD is create, read, update, and Delete. It allows users to interact with the data.
 13. What does HTTP stand for? 
+    * Hyper Text Transfer Protocol
 14. What are the two ways to interpolate Ruby in an ERB view template? What's the difference between these two ways?
 15. What's an ORM?
+    * Object Relational Map
 16. What's the most commonly used ORM in ruby (Sinatra & Rails)?
 17. Let's say we have an application with restaurants. There are seven verb + path combinations necessary to provide full CRUD functionality for our restaurant application. List each of the seven combinations, and explain what each is for.
+    * get '/restraunts' - is asking for the home restraunts_index
+    * get '/restraunts/:id - asking for specific restraunt
+    * get '/restraunts/:id/edit - asking for the edit page
+    * put '/restraunts/:id - updates single restraunt
+    * delete '/restraunts/:id - deletes single restraunt
+    * get '/restraunts/new - returns new restraunt form 
+    * get '/' returns the root directory which may be a dashboard

```

Comment:
Missing `post '/restaurants'` to create a new restaurant. I wouldn't consider the root part of these seven routes.